### PR TITLE
Quadlet system tests - fix socket notification

### PR DESCRIPTION
**Pull request by ygalblum**
In some tests use Notify=yes while in others remove the notification

Resolves: #18516

#### Does this PR introduce a user-facing change?
No

```release-note
None
```

### CODE REVIEW

File: test/system/252-quadlet.bats [hunk 1]
```diff
@@ -523,6 +524,7 @@ ContainerName=$NAME
 Image=$IMAGE
 Secret=$SECRET_NAME,type=env,target=MYSECRET
 Exec=sh -c "echo STARTED CONTAINER; echo "READY=1" | socat -u STDIN unix-sendto:\$NOTIFY_SOCKET; top"
+Notify=yes
```
Comment (vrothberg) [inline test/system/252-quadlet.bats:527]: Shouldn't this be `=true`?

Comment (Luap99) [reply test/system/252-quadlet.bats:527]: quadlet supports true, yes or 1 for booleans. I think yes is common for systemd units settings.

File: test/system/252-quadlet.bats [hunk 2]
```diff
@@ -467,7 +468,7 @@ EOF
 [Container]
 Image=$IMAGE
 SecurityLabelDisable=true
-Exec=sh -c "echo STARTED CONTAINER; echo "READY=1" | socat -u STDIN unix-sendto:\$NOTIFY_SOCKET; top"
+Exec=sh -c "echo STARTED CONTAINER; echo "READY=1" | top"
```
Comment (Luap99) [inline test/system/252-quadlet.bats:471]: this also should remove the `echo "READY=1" |` part

File: test/system/252-quadlet.bats [hunk 3]
```diff
@@ -494,7 +495,7 @@ Image=$IMAGE
 SecurityLabelType=spc_t
 SecurityLabelLevel=s0:c100,c200
 SecurityLabelFileType=container_ro_file_t
-Exec=sh -c "echo STARTED CONTAINER; echo "READY=1" | socat -u STDIN unix-sendto:\$NOTIFY_SOCKET; top"
+Exec=sh -c "echo STARTED CONTAINER; echo "READY=1" | top"
```
Comment (Luap99) [inline test/system/252-quadlet.bats:498]: same here

File: test/system/252-quadlet.bats (hunk 1 above)
Comment (ygalblum) [reply test/system/252-quadlet.bats:527]: Yes, the convention in systemd unit files is to use `yes` or `no`

File: test/system/252-quadlet.bats (hunk 2 above)
Comment (ygalblum) [reply test/system/252-quadlet.bats]: Sorry, somehow I misread '|' as '||' and kept it

File: test/system/252-quadlet.bats (hunk 3 above)
Comment (ygalblum) [reply test/system/252-quadlet.bats]: Fixed

**Review by vrothberg — COMMENTED** [review]
LGTM
@Luap99 PTanotherL

**Pull request merged**